manifests/server/config.pp

```diff
@@ -8,6 +8,9 @@
   $group    = $dns::server::params::group,
 ) inherits dns::server::params {
 
+  $log_dir  = '/var/log/named'
```

Comment:
How about putting this as a param? That way it can be a variable that can be set?

Is it not /var/log/bind on debian systems?